### PR TITLE
fix query param bug in getStreamByUserId

### PR DESCRIPTION
**Pull request by zunderscore**
Type: Bugfix

Fixes: Discord

Fixes field name in `getStreamByUserId`

### CODE REVIEW

File: packages/api/src/api/helix/stream/HelixStreamApi.ts [hunk 1]
```diff
@@ -119,15 +119,9 @@ export class HelixStreamApi extends BaseApi {
...
 	 */
 	async getStreamByUserId(user: UserIdResolvable): Promise<HelixStream | null> {
-		const userId = extractUserId(user);
-		const result = await this._client.callApi<HelixPaginatedResponse<HelixStreamData>>({
-			url: 'streams',
-			type: 'helix',
-			userId,
-			query: createStreamQuery({ userName: userId })
-		});
+		const result = await this.getStreamsByUserIds([user]);
```
Comment (d-fischer) [inline packages/api/src/api/helix/stream/HelixStreamApi.ts:122]: You removed the `userId` association here. Please revert.

**Review by d-fischer — CHANGES_REQUESTED** [review]

File: packages/api/src/api/helix/stream/HelixStreamApi.ts (hunk 1 above)
Comment (zunderscore) [reply packages/api/src/api/helix/stream/HelixStreamApi.ts:122]: When this gets passed to `getStreamsByUserIds`, is the user ID not extracted and mapped to the `userId` parameter (see line 111)?

Comment (d-fischer) [reply packages/api/src/api/helix/stream/HelixStreamApi.ts:122]: Yes, but that does not pass the user ID through to the `callApi` call, which is intentionally overwritten here to allow for sending the request with the explicitly given user's token.

Comment (zunderscore) [reply packages/api/src/api/helix/stream/HelixStreamApi.ts:122]: So then by that logic, `getStreamByUserName` is currently incorrectly written, correct? Or are you saying that this particular method should be inconsistent with the 3 others above it?

Comment (d-fischer) [reply packages/api/src/api/helix/stream/HelixStreamApi.ts:122]: By name is different, because it can by design not be used to identify a token.

Comment (d-fischer) [reply packages/api/src/api/helix/stream/HelixStreamApi.ts:122]: To clarify, the parameter that is relevant is the one in line 126, not the one in line 127.

Comment (zunderscore) [reply packages/api/src/api/helix/stream/HelixStreamApi.ts]: Reverted.

**Review by d-fischer — APPROVED** [review]

**Pull request merged**